ipa-core/Cargo.toml

```diff
@@ -135,7 +140,7 @@ sha2 = "0.10"
 shuttle-crate = { package = "shuttle", version = "0.6.1", optional = true }
 thiserror = "1.0"
 time = { version = "0.3", optional = true }
-tokio = { version = "1.35", features = ["fs", "rt", "rt-multi-thread", "macros"] }
+tokio = { version = "1.35", features = ["fs", "rt", "rt-multi-thread", "macros", "tracing"] }
```

Comment:
Could there be a cost to enabling this feature?

Reply:
for sure, I am going to take it only if `tokio-console` is enabled